src/common/Tile/styled.js

```diff
@@ -47,6 +47,7 @@ export const Poster = styled.img`
   background: ${({ theme }) => theme.colors.Silver};
   object-position: center center;
   object-fit: cover;
+  flex-shrink: 0;
```

Comment:
Nie wiem czemu, ale u mnie co do tego plakatu nie ma żadnej różnicy, może dlatego, że testuję to w porównaniu z wersją gdzie jest już float zastosowany

Reply:
Też się zastanawiam co ma to na celu 🤔

Reply:
![movie-poster](https://user-images.githubusercontent.com/67596987/97804852-b38a1880-1c52-11eb-8d21-2281b1cf49f9.PNG)
u mnie bez tej zmiany wygląda to tak

Reply:
O! 🙄 

No to faktycznie przydatne :D